velox/docs/functions/spark/aggregate.rst

```diff
@@ -7,6 +7,14 @@ Aggregate functions operate on a set of values to compute a single result.
 General Aggregate Functions
 ---------------------------
 
+.. spark:function:: avg(x) -> double|real
```

Comment:
Is this accurate? It seems like Spark never returns 'real' for avg, but it may return decimal.

Reply:
Thanks for pointing out this. Fixed.